### ISSUE TITLE
deploy: Follow up items for Helm Chart

### ISSUE THREAD

**Issue by zvonkok** [kata-containers/kata-containers#9924]
We need additional features for the Helm Chart: 

- [ ] Add Helm chart publishing to release.yaml 
- [ ] Add Helm chart for each PR build
- [ ] Latest PR build will also upload a latest Helm chart. 
- [ ] We need a "place" where to publish our Helm charts. I would be nice to have one Helm chart per PR

**Comment by fidencio** [comment]
This may help with a place for the helmcharts: https://github.com/helm/chart-releaser-action

**Comment by zvonkok** [comment]
Yes and no, it will not be part of our release but a separate release entry. I tried it and I do not like it really. I have a gha that we can reuse for latest and releases.

**Comment by efficacy38** [comment]
After tracing the pipeline, I found it would release this helm chart as a tarball. If someone interested this please check [kata-deploy-3.14.0.tgz at release page](https://github.com/kata-containers/kata-containers/releases)

However, the problem is I can't find any helm repo or OCI repository about this chart, which would make it hard to integrate this helm chart into the gitops workflow. Adding an OCI helm chart repository would be the most straightforward to manage versioning. 

Pushing this helm chart to the OCI registry via the helm cli is relatively easy to manage; @zvonkok, what do you think about adding this solution?

```
helm push kata-deploy-3.14.0.tgz oci://docker pull ghcr.io/kata-containers/kata-deploy-helm-chart
```

**Comment by tylerauerbeck** [comment]
@zvonkok It looks like this still needed. Can you confirm? If so, happy to 👀

**Comment by zvonkok** [comment]
On my backlog currently updating some of the installation instructions, will follow up with this soon'ish.

**Comment by tylerauerbeck** [comment]
@zvonkok I had happened to be doing something similar in another project, so thought it may help to toss this out as a quick PR to address the `values.schema.json` issue mentioned here. Happy to chat / adapt if it's useful.

**Comment by fidencio** [comment]
@tylerauerbeck, please, open a PR and we will review it.
